UNRELEASED.md

```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fix selector import in `DataTable` and `Cell` ([#638](https://github.com/Shopify/polaris-react/pull/638))
```

Comment:
Fixed

Reply:
This feels like a great opportunity to use the code suggestion feature.

```suggestion
- Fixed selector import in `DataTable` and `Cell` ([#638](https://github.com/Shopify/polaris-react/pull/638))
```

Reply:
Oh right, I have to rewire my brain to remember to use it.